### PR TITLE
Examples for Slovene

### PR DESCRIPTION
**Pull request by orglce**
## Description
Added example sentences for Slovenian language 

### Types of change
Enhancement

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: spacy/lang/sl/examples.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Example sentences to test spaCy and its language models.
+
+>>> from spacy.lang.sl.examples import sentences
+>>> docs = nlp.pipe(sentences)
+"""
+
+
+sentences = [
+    "Apple načrtuje nakup britanskga startupa za 1 bilijon dolarjev",
+    "France Prešeren je umrl 8. februarja 1849 v Kranju"
```
Comment (adrianeboyd) [inline spacy/lang/sl/examples.py:11]:
```suggestion
    "France Prešeren je umrl 8. februarja 1849 v Kranju",
```

File: spacy/lang/sl/examples.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+"""
+Example sentences to test spaCy and its language models.
+
+>>> from spacy.lang.sl.examples import sentences
+>>> docs = nlp.pipe(sentences)
+"""
+
+
+sentences = [
+    "Apple načrtuje nakup britanskga startupa za 1 bilijon dolarjev",
```
Comment (adrianeboyd) [inline spacy/lang/sl/examples.py:10]: I run examples through google translate just to double-check and it suggested that "britanskga" might be a misspelling of "britanskega". Could obviously be nonsense, but thought I'd ask to be sure?

**Comment by adrianeboyd** [comment]
Thanks for the PR!

File: spacy/lang/sl/examples.py (hunk 2 above)
Comment (orglce) [reply spacy/lang/sl/examples.py:10]: Oh yes it is a typo. "britanskega" is the right spelling. Thanks for checking.

**Comment by adrianeboyd** [comment]
Thanks for the fixes!

**Pull request merged**